deploy/examples/create-external-cluster-resources.py

```diff
@@ -962,9 +982,18 @@ def get_rbd_node_caps_and_entity(self):
                     "mandatory flag not found, please set the '--k8s-cluster-name' flag"
                 )
             entity = self.get_entity(
-                entity, rbd_pool_name, alias_rbd_pool_name, k8s_cluster_name
+                entity,
+                rbd_pool_name,
+                alias_rbd_pool_name,
+                k8s_cluster_name,
+                rados_namespace,
             )
-            caps["osd"] = f"profile rbd pool={rbd_pool_name}"
+            if rados_namespace != "":
+                caps[
+                    "osd"
+                ] = f"profile rbd pool={rbd_pool_name} namespace={rados_namespace}"
+            else:
+                caps["osd"] = f"profile rbd pool={rbd_pool_name}"
```

Comment:
It is the black formatter we use to format python code, 
Actually there is a limit to a line characters in a line so that why I think it lints like this